conf.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": "4.0",
-    "symfony_api_url": "https://api.symfony.com",
+    "symfony_api_url": "https://api.symfony.com/%s",
     "php_doc_url": "https://secure.php.net/manual/en",
     "symfony_doc_url": "https://symfony.com/doc/%s"
```

Comment:
In general, this all kinda looks like config that can be hardcoded - these URLs would never change, right?